### PR TITLE
Loyalty card updates

### PR DESCRIPTION
**Pull request by mabdullah12**

### CODE REVIEW

**Review by jkaliszuk — CHANGES_REQUESTED** [review]

File: src/main/java/io/voucherify/client/model/loyalties/AddBalance.java [hunk 1]
```diff
@@ -15,4 +15,6 @@
 public class AddBalance {
 
   private Long points;
+  private String expiration_date;
+  private String expiration_type;
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/model/loyalties/AddBalance.java:19]: Please rename variables to camelCase notation and add @JsonProperty annotations. Make enum for `expirationType` with elements `PROGRAM_RULES, CUSTOM_DATE, NON_EXPIRING` according to https://docs.voucherify.io/reference/add-loyalty-card-balance

File: src/main/java/io/voucherify/client/model/voucher/LoyaltyCard.java [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.ToString;
 
+import java.sql.Date;
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/model/voucher/LoyaltyCard.java:11]: Use `import java.util.Date;` like everywhere else in project.

File: src/main/java/io/voucherify/client/api/VoucherifyApi.java [hunk 3]
```diff
@@ -468,6 +468,11 @@ Call<io.voucherify.client.model.loyalties.response.AddBalanceResponse> addLoyalt
       @Path("memberId") String memberId,
       @Body io.voucherify.client.model.loyalties.AddBalance addBalance);
 
+  @POST("loyalties/members/{memberId}/balance")
+  Call<io.voucherify.client.model.loyalties.response.AddBalanceResponse> addLoyaltyBalance(
+          @Path("memberId") String memberId,
+          @Body io.voucherify.client.model.loyalties.AddBalance addBalance);
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/api/VoucherifyApi.java:474]: Fix tabulation

File: src/main/java/io/voucherify/client/module/LoyaltiesModule.java [hunk 4]
```diff
@@ -123,6 +123,11 @@ public AddBalanceResponse addLoyaltyCardBalance(
     return executeSyncApiCall(api.addLoyaltyBalance(id, memberId, addBalance));
   }
 
+    public AddBalanceResponse addLoyaltyCardBalance(
+            String memberId, AddBalance addBalance) {
+        return executeSyncApiCall(api.addLoyaltyBalance(memberId, addBalance));
+    }
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/module/LoyaltiesModule.java:129]: Fix tabulation

**Review by jkaliszuk — CHANGES_REQUESTED** [review]

File: src/main/java/io/voucherify/client/model/loyalties/BalanceExpirationType.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package io.voucherify.client.model.loyalties;
+
+public enum BalanceExpirationType {
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/model/loyalties/BalanceExpirationType.java:3]: Unused enum.

File: src/main/java/io/voucherify/client/model/loyalties/AddBalance.java [hunk 6]
```diff
@@ -15,4 +16,10 @@
 public class AddBalance {
 
   private Long points;
+
+  @JsonProperty("expiration_date")
+  private String expirationDate;
+
+  @JsonProperty("expiration_type")
+  private String expirationType;
```
Comment (jkaliszuk) [inline src/main/java/io/voucherify/client/model/loyalties/AddBalance.java:24]: Use BalanceExpirationType here

**Review by jkaliszuk — CHANGES_REQUESTED** [review]

File: src/main/java/io/voucherify/client/model/loyalties/AddBalance.java (hunk 6 above)
Comment (mabdullah12) [reply src/main/java/io/voucherify/client/model/loyalties/AddBalance.java]: My bad. I missed it. Apologies. It's done now.

**Review by jkaliszuk — APPROVED** [review]

**Pull request merged**